### PR TITLE
Allow to publish community.dns

### PR DESCRIPTION
**Pull request by felixfontein**
Follow-up to #793

CC @gundalow

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -198,6 +198,10 @@
     name: github.com/ansible-collections/community.digitalocean
     templates:
       - publish-to-galaxy-3pci
+- project:
```
Comment (gundalow) [inline zuul.d/projects.yaml:201]: I wonder if you need to rebase this PR?

Comment (felixfontein) [reply zuul.d/projects.yaml:201]: I did, this branch contains the merge commit. Maybe Zuul takes a little bit of time to reload the config and I created this PR too quick... :)

**Comment by felixfontein** [comment]
recheck

**Comment by felixfontein** [comment]
recheck

**Comment by felixfontein** [comment]
CI passed :)

File: zuul.d/projects.yaml (hunk 1 above)
Comment (pabelanger) [reply zuul.d/projects.yaml:201]: There is a promote job that runs, which lays down the updated zuul tenant file, then reload the schedule. Once that is finished, this PR would have worked.

**Pull request merged**